radar-upload-frontend/src/axios/__tests__/file.spec.js

```diff
@@ -205,6 +205,6 @@ describe('axios/file', () => {
     const fileName = 'name';
     await fileAPI.download({ recordId, fileName });
```

Comment:
I think call the download API this way still loading the file into the memory before being able to save it